junit5/README.md

```diff
@@ -396,3 +546,10 @@ class ExplicitParamInjectionTest {
 
 As you might know, if you want to inject a bean where you would normally not use any qualifier, you can do that using `@Default` qualifier (as shown in the code above).
 This is in accordance with CDI specification, feel free to [read more about it](http://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#builtin_qualifiers).
+
+#### Flat Deployment
```

Comment:
I moved flat deployment option to additional config section but now that I think of it, it only makes sense with enabled discovery, which can only happen in `WeldJunit5Extension`, right?

Reply:
Well, it seems that `WeldJunit5AutoExtension` allows you to use `@WeldSetup WeldInitiator` to configure the container instance. So if this is intended I think it's ok.